### PR TITLE
[Hub Generated] Review request for Microsoft.DigitalTwins to add version preview/2020-03-01-preview

### PR DESCRIPTION
**Pull request by sup-s**
If you are a MSFT employee you can view your [work branch via this link](https://portal.azure-devex-tools.com/branch/sup-s/azure-rest-api-specs/dev%2Fsupsanka%2FdigitalTwins...master/).

### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by sup-s** [comment]
The swagger has been approved by ARM in the private repo. See: Azure/azure-rest-api-specs-pr#999.
In this PR, I have
1. Changed API version to 2020-03-01-preview (was: 2020-02-26-preview)
2. Addressed feedback on private repo PR.

**Review by ryansbenson — COMMENTED** [review]
Approved from ARMs side

**Review by erich-wang — APPROVED** [review]

File: specification/digitaltwins/resource-manager/readme.go.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  clear-output-folder: true
+```
+
+### Tag: package-2020-03-01-preview and go
+
+These settings apply only when `--tag=package-2020-03-01-preview --go` is specified on the command line.
+Please also specify `--go-sdks-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+```yaml $(tag) == 'package-2020-03-01-preview' && $(go)
+namespace: Microsoft.DigitalTwins
```
Comment (ArcturusZhang) [inline specification/digitaltwins/resource-manager/readme.go.md]: Please change this to `digitaltwins`

File: specification/digitaltwins/resource-manager/readme.go.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+output-folder: $(go-sdks-folder)/digitaltwins/Generated
```
Comment (ArcturusZhang) [inline specification/digitaltwins/resource-manager/readme.go.md]: Please change the output folder to 
`$(go-sdk-folder)/services/preview/$(namespace)/mgmt/2020-03-01-preview/$(namespace)`

**Review by ArcturusZhang — CHANGES_REQUESTED** [review]

File: specification/digitaltwins/resource-manager/readme.go.md (hunk 1 above)
Comment (ArcturusZhang) [reply specification/digitaltwins/resource-manager/readme.go.md]: The namespace should be `digitaltwins`, please change the namespace.

**Review by ArcturusZhang — CHANGES_REQUESTED** [review]

**Comment by ArcturusZhang** [comment]
Hi @sup-s please change the namespace in `readme.go.md`. Despite the CI is not reporting errors now, the go SDK will not compile when the namespace contains dot or up-case letters.

**Review by ArcturusZhang — APPROVED** [review]

**Pull request merged**